src/main/java/org/osc/controller/nuage/api/NuageSdnRedirectionApi.java

```diff
@@ -186,6 +186,11 @@ public InspectionHookElement getInspectionHook(String inspectionHookId) throws E
     }
 
     @Override

```

Comment:
Is there a corresponding change in api?

Reply:
Yes, this will make sure this build does not fail. The api has been merged, you reviewed it :)